docs/source/installation/setting-up-an-environment.rst

```diff
@@ -118,7 +118,7 @@ Choosing an installation method
 -------------------------------
 Once you have a virtual environment up and running, choose from one of the following 4 installation methods:
 
-1. :doc:`regular-pypi`,
-2. :doc:`regular-anaconda`,
-3. :doc:`development-pypi`, or
-4. :doc:`development-anaconda`.
+1. :doc:`regular-pip`,
+2. :doc:`regular-conda`,
+3. :doc:`developer-pip`, or
+4. :doc:`developer-conda`.
```

Comment:
Technically the conda developer install also uses `pip`. Perhaps instead of having it be `-<package manager>` it should be `-<Python distribution>`? So `*-cpython` and `*-anaconda`?

Reply:
This was a typo (these links don't work), and should read:
```rst
1. :doc:`regular-pypi`,
2. :doc:`regular-anaconda`,
3. :doc:`developer-pypi`, or
4. :doc:`developer-anaconda`.
```